src/api/cpp/cvc5_types.cpp

```diff
@@ -30,6 +30,7 @@ std::ostream& operator<<(std::ostream& out, UnknownExplanation e)
       out << "REQUIRES_FULL_CHECK";
       break;
     case UnknownExplanation::INCOMPLETE: out << "INCOMPLETE"; break;
+    case UnknownExplanation::UNSOUND: out << "UNSOUND"; break;
```

Comment:
Maybe we should go with "REFUTATION_UNSOUND"? And conversely with "SOLUTION_UNSOUND".

Reply:
On second thought I will leave this enum as is, where incomplete = model unsound or refutation unsound.